packages/api/src/useFetch.ts

```diff
@@ -13,7 +13,10 @@ const useFetch = <T extends TSocketEndpointNames>(name: T, ...props: TSocketAcce
     const payload = prop && 'payload' in prop ? (prop.payload as TSocketRequestPayload<T>) : undefined;
     const options = prop && 'options' in prop ? (prop.options as TSocketRequestQueryOptions<T>) : undefined;
 
-    return useQuery<TSocketResponseData<T>, unknown>(getQueryKeys(name, payload), () => send(name, payload), options);
+    return useQuery<TSocketResponseData<T>, unknown>(getQueryKeys(name, payload), () => send(name, payload), {
```

Comment:
Or maybe we can write like this in `APIProvider.tsx`:

`const queryClient = new QueryClient({ defaultOptions: { queries: { refetchOnWindowFocus: false } } });`